### ISSUE TITLE
Multiple package in same repository triggers not installable errors

### ISSUE THREAD

**Issue by canatella** [purcell/package-lint#172]
So as per https://github.com/purcell/package-lint/issues/125#issuecomment-602902003, I splitted my package in multiple ones, they are all in the same repo. There are dependencies between those packages. I have a base package `xwwp-follow-link`, and one package for helm that depends via the `Package-Requires` tag on the base one. When I run package lint, now I get a 

```
xwwp-follow-link-helm.el:8:38: error: Package xwwp-follow-link is not installable.
```
Which makes sense, as the package is not in any archive, and even if it were it would not be the right version. The thing is I splitted the package to avoid the `with-eval-after-load' warning and now find myself with a package not installable error ;)

**Comment by purcell** [comment]
Ultimately this warning will only last until `xwwp-follow-link` is in MELPA, so I don't consider it a big issue, but I can see how it could frustrate initial set-up of strict `package-lint` in CI, for example.

I can't offer a great workaround for that initial bootstrap period in which you're writing multiple new interdependent packages simultaneously, sorry, but I'll bear this in mind.

Rest assured you're doing the right thing. :-)

**Comment by canatella** [comment]
All right then, thanks for you help.
